sage-ios/SAGE/SAGE/NotificationConstants.swift

```diff
@@ -16,5 +16,6 @@ struct NotificationConstants {
     static let editAnnouncementKey: String = notificationBaseName + "editAnnouncement"
     static let addCheckinKey: String = notificationBaseName + "addCheckin"
     static let editProfileKey: String = notificationBaseName + "editProfile"
-    
+    static let addVerifiedCheckinKey: String = notificationBaseName + "addVerifiedCheckin"
+    static let addUnverifiedCheckinKey: String = notificationBaseName + "addUnverifiedCheckin"
```

Comment:
oh didnt even realize addCheckinKey was there. it seems like its not used anywhere, so im gonna delete that string const actaully